torchbenchmark/util/framework/gnn/requirements.txt

```diff
@@ -0,0 +1 @@
+torch_geometric @ git+https://github.com/pyg-team/pytorch_geometric.git@cabcd4097442ba60aa1efa11e1619dd9bb8fb527
```

Comment:
Is it suitable to fix a certain commit here? Why not use `pip install pyg-nightly`?

Reply:
I figured fixing a commit would make things more stable as we wouldn't arbitrarily break as people pushed things to the repo. Can't use the nightly as it's incompatible with PyTorch nightly